### PR TITLE
Message auditing

### PR DESCRIPTION
**Pull request by paulofariadias**

### CODE REVIEW

**Comment by paulofariadias** [comment]
Created Message Auditing topic based on the [KB article](https://desk.rocket.chat/portal/kb/articles/message-auditing-panel). Added setting preferences. Added screenshots.

**Comment by paulofariadias** [comment]
Fixed trailing spaces.

File: administrator-guides/message-auditing/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Message Auditing Panel
```
Comment (MartinSchoeler) [inline administrator-guides/message-auditing/README.md:1]: Can you add this file to the [table of contents](https://github.com/RocketChat/docs/blob/master/_data/toc.yml)?

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

File: administrator-guides/message-auditing/README.md (hunk 1 above)
Comment (paulofariadias) [reply administrator-guides/message-auditing/README.md:1]: Updated TOC.

**Review by MartinSchoeler — APPROVED** [review]

**Pull request merged**